projects/1.16/assets/mr-pineapples-food-mod/pinesfoodmod/lang/zh_cn.json

```
@@ -1 +1,110 @@
-{}
\ No newline at end of file
+{
+    "item.pinesfoodmod.green_apple": "青苹果",
+	"block.pinesfoodmod.corn": "玉米",
+	"block.pinesfoodmod.grapes": "葡萄",
+	"block.pinesfoodmod.pepper_bush": "辣椒", 
+	"block.pinesfoodmod.blueberry_bush": "蓝莓",
+	"block.pinesfoodmod.cherry_bush": "樱桃",
+	"block.pinesfoodmod.chocolate_cake": "巧克力蛋糕",
+	"block.pinesfoodmod.glazed_chocolate_cake": "巧克力脆皮蛋糕",
+	"block.pinesfoodmod.decorated_chocolate_cake": "巧克力装饰蛋糕",
+	"block.pinesfoodmod.birthday_cake": "生日蛋糕",
+	"block.pinesfoodmod.red_velvet_cake": "红丝绒蛋糕",
+	"item.pinesfoodmod.bottle_of_chocolate_sauce": "瓶装巧克力酱",
+	"item.pinesfoodmod.wine": "葡萄酒",
+	"item.pinesfoodmod.milk_bottle": "瓶装牛奶",
+	"item.pinesfoodmod.orange": "橙子",
+	"item.pinesfoodmod.wrapper": "素色包装袋",
+	"item.pinesfoodmod.white_chocolate_bar": "白色包装巧克力棒",
+	"item.pinesfoodmod.orange_chocolate_bar": "橙色包装巧克力棒",
+	"item.pinesfoodmod.magenta_chocolate_bar": "品红色包装巧克力棒",
+	"item.pinesfoodmod.light_blue_chocolate_bar": "淡蓝色包装巧克力棒",
+	"item.pinesfoodmod.yellow_chocolate_bar": "黄色包装巧克力棒",
+	"item.pinesfoodmod.lime_chocolate_bar": "黄绿色包装巧克力棒",
+	"item.pinesfoodmod.pink_chocolate_bar": "粉红色包装巧克力棒",
+	"item.pinesfoodmod.gray_chocolate_bar": "灰色包装巧克力棒",
+	"item.pinesfoodmod.light_gray_chocolate_bar": "淡灰色包装巧克力棒",
+	"item.pinesfoodmod.cyan_chocolate_bar": "青色包装巧克力棒",
+	"item.pinesfoodmod.purple_chocolate_bar": "紫色包装巧克力棒",
+	"item.pinesfoodmod.blue_chocolate_bar": "蓝色包装巧克力棒",
+	"item.pinesfoodmod.brown_chocolate_bar": "棕色包装巧克力棒",
+	"item.pinesfoodmod.green_chocolate_bar": "绿色包装巧克力棒",
+	"item.pinesfoodmod.red_chocolate_bar": "红色包装巧克力棒",
+	"item.pinesfoodmod.black_chocolate_bar": "黑色包装巧克力棒",
+	
+	"block.pinesfoodmod.brick_pizza_oven": "红砖披萨烤炉",
+	"block.pinesfoodmod.pizza_oven": "圆石披萨烤炉",
+	"block.pinesfoodmod.pizza_base": "披萨饼底",
+	"block.pinesfoodmod.pizza_raw_base": "生披萨饼底",
+	"block.pinesfoodmod.pizza_box": "披萨盒",
+	"item.pinesfoodmod.cardboard": "卡纸板",
+	
+	
+	"item.pinesfoodmod.flour": "面粉",
+	"item.pinesfoodmod.bag_of_flour": "一袋面粉",
+	"block.pinesfoodmod.olives": "橄榄",
+	"item.pinesfoodmod.olive_oil": "橄榄油",
+	"item.pinesfoodmod.salt": "盐",
+	"item.pinesfoodmod.yeast": "酵母",
+	"item.pinesfoodmod.pepperoni": "意大利红肠",
+	"block.pinesfoodmod.tomato": "番茄",
+	"block.pinesfoodmod.garlic": "大蒜",
+	"block.pinesfoodmod.onion": "洋葱",
+	"item.pinesfoodmod.pineapple": "菠萝",
+	"item.pinesfoodmod.bacon": "培根",
+	"item.pinesfoodmod.cooked_bacon": "熟培根",
+	"item.pinesfoodmod.mozzarella_cheese": "马苏里拉奶酪",
+	"item.pinesfoodmod.cheddar_cheese": "切达奶酪",
+	"item.pinesfoodmod.mature_cheddar_cheese": "成熟的切达奶酪",
+	"item.pinesfoodmod.ricotta_cheese": "里考塔奶酪",

```

```suggestion
	"item.pinesfoodmod.mature_cheddar_cheese": "熟成的切达奶酪",
	"item.pinesfoodmod.ricotta_cheese": "里科塔奶酪",
```
里科塔这个纯属个人听得多了（）
如果这里更改则全文件同步

---

ricotta这个是按维基百科来的来着...